server/app/api/v1/endpoints/subscriptions.py

```
@@ -0,0 +1,2016 @@
+            "Failed to reactivate subscription",
+            extra={
+                "subscription_id": str(subscription_id),
+                "organization_id": str(organization.id),
+                "error": str(e),
+            },
+            exc_info=True,
+        )
+        raise HTTPException(
+            status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
+            detail="Failed to reactivate subscription. Please try again or contact support.",
+        )
```

Ruff (0.14.14)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)